assets/src/scss/layout/_navbar.scss

```diff
@@ -70,6 +70,14 @@ $min-height: 40px;
 $navbar-default-height: 60px;
 
 .top-navigation {
+  _-- {
+    background: transparentize($dark-blue, 0.2);
+    border-bottom-width: 0;
```

Comment:
Had to add these because the climate theme has a white border. We could also check if we can remove it from that theme. If I remember correctly it was added because the color of the fixed navbar didn't contrast enough with the content.